core/templates/components/state-editor/state-skill-editor/state-skill-editor.component.spec.ts

```diff
@@ -197,7 +197,11 @@ describe('State Skill Editor Component', () => {
     fixture.detectChanges();
     componentInstance.untriagedSkillSummaries = [];
     urlInterpolationService = TestBed.inject(UrlInterpolationService);
-    mockNgbModal = (TestBed.inject(NgbModal) as unknown) as MockNgbModal;
+    // This throws "Type NgbModel is not assignable to type
+    // 'Model'." We need to suppress this error
+    // because of the need to test validations.
+    // @ts-ignore
+    mockNgbModal = (TestBed.inject(NgbModal)) as MockNgbModal;
```

Comment:
```suggestion
    // This throws "Type NgbModal is not assignable to type
    // 'Modal'." We need to suppress this error
    // because of the need to test validations.
    // @ts-ignore
    mockNgbModal = TestBed.inject(NgbModal) as MockNgbModal;
```
?

Also, same comments as any above ignores.

Reply:
Done